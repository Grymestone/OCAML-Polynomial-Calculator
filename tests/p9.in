(2a^2)^20
