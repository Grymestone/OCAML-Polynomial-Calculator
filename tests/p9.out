((2*(a)^2))^20
1048576x^40